### PR TITLE
Enable ValueType testing on all platforms

### PR DESCRIPTION
**Pull request by tajila**
Enable ValueType testing on all platforms

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
@AdamBrousseau please review these changes

**Comment by tajila** [comment]
I guess it will use eclipse-openj9/openj9 rather than this PR for PR builds

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy [hunk 1]
```diff
@@ -153,7 +153,21 @@ SPECS = ['ppc64_aix' : CURRENT_RELEASES,
          's390x_linux_aot'    : CURRENT_RELEASES,
          'x86-64_linux_aot'   : CURRENT_RELEASES,
          'x86-64_mac_aot'     : CURRENT_RELEASES,
-         'x86-64_windows_aot' : CURRENT_RELEASES]
+         'x86-64_windows_aot' : CURRENT_RELEASES
```
Comment (AdamBrousseau) [inline buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy:156]: Need a comma `,` at end of this line.

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy [hunk 2]
```diff
@@ -262,7 +276,21 @@ SHORT_NAMES = ['all' : ['ppc64le_linux','s390x_linux','x86-64_linux','ppc64_aix'
             'zlinuxaot' : ['s390x_linux_aot'],
             'xlinuxaot' : ['x86-64_linux_aot'],
             'osxaot' : ['x86-64_mac_aot'],
-            'winaot' : ['x86-64_windows_aot']]
+            'winaot' : ['x86-64_windows_aot']
```
Comment (AdamBrousseau) [inline buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy:279]: Need a comma `,` at end of this line.

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy [hunk 3]
```diff
@@ -153,7 +153,21 @@ SPECS = ['ppc64_aix' : CURRENT_RELEASES,
...
-         'x86-64_windows_aot' : CURRENT_RELEASES]
+         'x86-64_windows_aot' : CURRENT_RELEASES
+         'ppc64le_linux_valhalla'   : ['next'],
+         'ppc64le_linux_vt_standard' : ['next'],
+         'aarch64_linux_valhalla'   : ['next'],
+         'aarch64_linux_vt_standard' : ['next'],
+         'x86-64_mac_valhalla'   : ['next'],
+         'x86-64_mac_vt_standard' : ['next'],
+         's390x_zos_valhalla'   : ['next'],
+         's390x_zos_vt_standard' : ['next'],
```
Comment (AdamBrousseau) [inline buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy:164]: I think we only have 11 for z/os at the moment so I'm not sure if these 2 lines are req'd. Won't hurt anything but may not work if you try it.

**Review by AdamBrousseau — CHANGES_REQUESTED** [review]
Your PR should work once those commas are fixed.

**Comment by tajila** [comment]
Jenkins test sanity,extended plinuxval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended zlinuxval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended alinuxval jdknext

**Comment by tajila** [comment]
Jenkins test sanity plinuxvalst jdknext

**Comment by tajila** [comment]
Jenkins test sanity zlinuxvalst jdknext

**Comment by tajila** [comment]
Jenkins test sanity alinuxvalst jdknext

**Review by AdamBrousseau — APPROVED** [review]

**Review by AdamBrousseau — COMMENTED** [review]
I approved and then noticed the build failures. I'll have to take a more in depth look to see why those errors are there.

**Comment by tajila** [comment]
I think the s390x failure is intermittent

**Comment by tajila** [comment]
Jenkins test sanity zlinuxvalst jdknext

**Comment by tajila** [comment]
Jenkins test sanity aixvalst jdknext

**Comment by tajila** [comment]
Jenkins test sanity winvalst jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended osxval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended aixval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended macval jdknext

**Comment by AdamBrousseau** [comment]
@tajila I think you just need to rebase your branch.
The aarch spec now builds in docker and should look like this
```
aarch64_linux:
  extends: ['boot_jdk_default', 'debuginfo', 'openjdk_reference_repo', 'openssl']
  boot_jdk:
    arch: 'aarch64'
    os: 'linux'
  release:
    all: 'linux-aarch64-server-release'
    8: 'linux-aarch64-normal-server-release'
    11: 'linux-aarch64-normal-server-release'
  node_labels:
    build: 'ci.role.build && hw.arch.aarch64 && sw.tool.docker'
    docker_image:
      11: 'adoptopenjdk/centos7_build_image@sha256:b2cc072a5c6196461a0993c6817d7540b8c8fcbce40f63cd5bb49eb6d7d61d89'
  build_env:
    vars: 'CC=/usr/local/gcc/bin/gcc-7.5 CXX=/usr/local/gcc/bin/g++-7.5'
```

**Comment by tajila** [comment]
Jenkins test sanity winvalst jdknext

**Comment by tajila** [comment]
Jenkins test sanity alinuxvalst jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended alinux jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended alinuxval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended winval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended alinuxval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended winval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended aixval jdknext

**Comment by tajila** [comment]
Jenkins test sanity,extended osxval jdknext

**Comment by tajila** [comment]
We will need to fix some compilation issues on some of the platforms, but looks like the builds work.

**Comment by tajila** [comment]
@pshipton please review/merge these changes

**Review by pshipton — APPROVED** [review]

**Pull request merged**